Quaver.Shared/Screens/Edit/EditScreenView.cs

```diff
@@ -131,6 +133,12 @@ public EditScreenView(Screen screen) : base(screen)
             EditScreen.BackgroundBrightness.ValueChanged += OnBackgroundBrightnessChanged;
             BackgroundHelper.Loaded += OnBackgroundLoaded;
             Footer.Parent = Container;
+            
+            ThreadScheduler.Run(() =>
```

Comment:
You don't need to run this on another thread. The Load method already does this and has cancellation logic.

Reply:
I wasn't sure whether to create a thread or not, but I decided to do it because of the comment written [here](https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Screens/Selection/UI/FilterPanel/SelectFilterPanel.cs#L342). If you can confirm that it's not needed in this case, I will remove it.

Reply:
I see. I think it's fine then.